File: LC/java/Remove Element.java
public class Solution {
    public int removeElement(int[] A, int elem) {
        int f = 0, e = A.length-1;
        if(A.length==0){
            return 0;
        }
        
        while(f<=e){
            if(A[f]==elem){
                A[f] = A[e];
                e--;
            }
            else{
                f++;
            }
        }
        return e+1;
    }
}

File: LC/python/Remove Element.py
class Solution:
    # @param    A       a list of integers
    # @param    elem    an integer, value need to be removed
    # @return an integer
    def removeElement(self, A, elem):
        if len(A) == 0:
            return 0
        else:
            while elem in A:
                A.remove(elem)
            return len(A)
        
